shell/src/test/java/tachyon/shell/TFsShellTest.java

```diff
@@ -260,27 +275,20 @@ public void fileinfoNotExistTest() throws IOException {
     Assert.assertEquals(-1, ret);
   }
 
-  @Test
-  public void fileinfoTest() throws IOException {
-    int fileId = TachyonFSTestUtils.createByteFile(mTfs, "/testFile", WriteType.MUST_CACHE, 10);
-    mFsShell.run(new String[] {"fileinfo", "/testFile"});
-    TachyonFile tFile = mTfs.getFile(new TachyonURI("/testFile"));
-    Assert.assertNotNull(tFile);
-    List<ClientBlockInfo> blocks = mTfs.getFileBlocks(fileId);
-    String[] commandParameters = new String[3 + blocks.size()];
-    commandParameters[0] = "fileinfo";
-    commandParameters[1] = "/testFile";
-    commandParameters[2] = String.valueOf(fileId);
-    Iterator<ClientBlockInfo> iter = blocks.iterator();
-    int i = 3;
-    while (iter.hasNext()) {
-      commandParameters[i ++] = iter.next().toString();
-    }
-    Assert.assertEquals(getCommandOutput(commandParameters), mOutput.toString());
-  }
-
-  private File generateFileContent(String path, byte[] toWrite) throws IOException,
-      FileNotFoundException {
+  /*
+   * @Test public void fileInfoTest() throws IOException { TachyonFile file =
```

Comment:
Should there be a TODO describing what to do with this test ... also the formatting seems off